src/http/http.cpp

```diff
@@ -17,15 +17,19 @@ std::vector<std::thread> workers = {};
 
 } // namespace
 
-void tfs::http::start(std::string_view address, unsigned short port /*= 8080*/, int threads /*= 1*/)
+void tfs::http::start(bool bindOnlyOtsIP, std::string_view otsIP, unsigned short port /*= 8080*/, int threads /*= 1*/)
 {
 	if (port == 0 || threads < 1) {
 		return;
 	}
 
-	fmt::print(">> Starting HTTP server on {:s}:{:d} with {:d} threads.\n", address, port, threads);
+	asio::ip::address address = asio::ip::address_v6::any();
+	if (bindOnlyOtsIP) {
+		address = asio::ip::make_address(otsIP);
+	}
+	fmt::print(">> Starting HTTP server on {:s}:{:d} with {:d} threads.\n", address.to_string(), port, threads);
```

Comment:
Setting `bindOnlyGlobalAddress` is exactly the same as setting `ip` to `0.0.0.0` (you can see the code inside `asio::ip::address_v6::any`, the default constructor zeroes the IP fields). Just changing the comment in `config.lua` to use `0.0.0.0` when using Docker really should suffice